torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -378,8 +378,12 @@ def affine_segmentation_mask(
 
 
 def _convert_fill_arg(fill: Optional[Union[int, float, Sequence[int], Sequence[float]]]) -> Optional[List[float]]:
+    # Fill = 0 is not equivalent to None, https://github.com/pytorch/vision/issues/6517
```

Comment:
Stable is using `fill=None` everywhere where I could spotted the difference, so that's why there is no change there.
However, we may want to fix the current inconsistency in stable with `fill=None` != `fill=0` (#6517).